pages/1.11/release-notes/1.11.5/index.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.11.5
+title: Release Notes for 1.11.5
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.11.5
+---
+
+DC/OS 1.11.5 was released on September 4, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.5/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+# Issues Fixed in DC/OS 1.11.5
+
```

Comment:
Gentle reminder to add in a section here to prepare for the link to the Service Cloud announcement featuring the Customer Advisory that will come from Razi (Networking PM)/Jena (Support)/me.

Reply:
##Customer Advisory
[loren ipsum]

##Frontend
etc...

Reply:
Done.